Adding location (boulder, co) and image (flatirons)

nquarder opened the pull request:
@lwasser here i am trying to do a pull request to the earthlab-education/ea-bootcamp-hometowns repo. This is related to Issue #1, I think?   -NQ

nquarder committed:
adding location and image

nquarder committed:
Delete boulder-colorado.ipynb

lwasser commented:
great start at this @nquarder ! it looks like you have an extra file in your pr - https://github.com/earthlab-education/ea-bootcamp-hometowns/pull/63/files notice there are 4 files instead of 3! you can delete that extra file without a name locally and update your fork to fix this! if you have any questions say the word!!

nquarder committed:
Delete .ipynb

nquarder committed:
added decorah iowa

nquarder commented:
okay @lwasser i deleted the extra un-named file and added an .ipynb for decorah, iowa to the iowa subdirectory in the united-state-of-america directory. This pr addresses issue #1.

lwasser commented:
nice work @nquarder !!! 🎆  !!!looks good!
